tools/autodeployment/server.js

```diff
@@ -0,0 +1,29 @@
+require('dotenv').config();
+const http = require('http');
+const createHandler = require('github-webhook-handler');
+const shell = require('shelljs');
+
+const { SECRET, REPO_NAME, DEPLOY_PORT } = process.env;
+
+const handler = createHandler({ path: '/', secret: SECRET });
+
+http.createServer((req, res) => {
+  handler(req, res, err => {
+    res.statusCode = 404
+    res.end('Page not found');
+  })
+}).listen(CD_PORT, () => {
+    console.log(`Server listening on port ${DEPLOY_PORT}`);
```

Comment:
We could do this later, but at some point we probably need to figure out how to get this to log somewhere we can go read what happened when things fail.